controller/value_resolver.rst

```diff
@@ -213,6 +213,83 @@ PSR-7 Objects Resolver:
     :class:`Psr\\Http\\Message\\RequestInterface` or :class:`Psr\\Http\\Message\\MessageInterface`.
     It requires installing :doc:`the PSR-7 Bridge </components/psr7>` component.
 
+Managing Value Resolvers
+------------------------
+
+For each argument, every resolver tagged with ``controller.argument_value_resolver``
+will be called until one provides a value. The order in which they are called depends
+on their priority. For example, the ``SessionValueResolver`` (priority 50) will be
```

Comment:
```suggestion
on their priority. For example, the ``SessionValueResolver`` (priority 50) will be
```

sometime due to new code addition, the priority changes (like for ex for event listener)
Is it desires to hardcode the priority here?
Cant we just only say that it is prior that

Reply:
Yes the idea is just that `SessionValueResolver` will be called before `DefaultValueResolver` since its priority is higher :+1: